### DISCUSSION TITLE
Problem - Command coming out along with the text

### DISCUSSION THREAD

**Discussion by leandrodrh** [GorvGoyl/ChatGPT-Writer#120] [Report Bugs and Issues]
In the text of the answer, the command that I put in "Enter your command for AI" appears. Until yesterday this did not happen. Is it a problem, or some setting that I activated? Can I use it without this happening?

Can any kind soul help me?

![Capturar](https://github.com/GorvGoyl/ChatGPT-Writer/assets/137516393/6bfbf10e-bd5a-4f17-9527-f1dcba6678a3)

**Comment by GorvGoyl** [comment]
Hi, There is an ongoing issue with response generation for free users and
we are actively working on a solution. We'll have it resolved as quickly as
we can.
